app/_src/quickstart/universal-demo.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Deploy Kuma on Universal
```

Comment:
@lahabana I added a quickstart for universal. Is everything on this page accurate? Specifically the deploy the demo section, I'm not sure if I got that right.